static/styles/popovers.css

```diff
@@ -165,6 +165,10 @@ ul {
     &.info_popover_actions i.fa-edit {
         vertical-align: middle;
     }
+
+    &.actions_popover a:focus {
+        outline: none;
+    }
```

Comment:
Can you post a screenshot showing why you made this change?

Reply:
Without this -> 
![image](https://user-images.githubusercontent.com/76561593/153939641-1417ac7a-6cca-4d8f-8ed0-47d18323d9ea.png)


With this -> 
![image](https://user-images.githubusercontent.com/76561593/153939525-036c3fa3-c9cf-47cb-8245-7fef6b08d419.png)

Reply:
@timabbott , I made this change to remove the blue border over the focused option in message-menu.

Reply:
But this also creates a doubt in mind regarding the difference between hover and focus on an element.

The above ss without the removal of outline on focus, makes it look different than the option being hovered.
and this change in CSS would make the option look same in both cases -> 

1.) Element being focused by the use of keyboard.
2.) Element is hovered upon, by the use of mouse.

Reply:
and i think we should follow the same approach as we're following in rest of our application for focusing over an element, so i'm reverting this change as in the whole application, we've the blue border outline on focus over a element.

Thanks @timabbott .